net/gatling/patches/020-configure_features.patch

```diff
@@ -0,0 +1,30 @@
+gatling: configure features
+
+Disables SMB-support, as only deprecated SMBv1 is supported,
+which most users wouldn't use anyway. Avoids potential
+vulnerabilities.
+
+Activates threaded opening: Thus gatling will open files in
+threads to open kernel I/O scheduling opportunities.
```

Comment:
Is this proposed to upstream?

Reply:
No, it isn't. The patch only tweaks Pre-processor directives in `gatling_features.h`. As far as I can see, this file is intended, to give the compiling user the ability to include or exclude functionality in the webserver. The corresponding webserver code is surrounded by `ifdefs`, so it can be activated/deactivated easily.

I consider that one mere configuration options for the user. I don't plan to bring them upstream, as poeple on x86-machines might have other demands on their special setups.